File: sdk/go/client/telemetry.go
package client

import (
	"log"
	"time"
)

func Track(
	event string,
	metadata map[string]interface{},
) {

	log.Printf(
		"[telemetry] %s %v %s",
		event,
		metadata,
		time.Now().UTC(),
	)
}

File: sdk/ java/src/main/java/com/aiplatform/sdk/Telemetry.java
package com.aiplatform.sdk;

import java.time.Instant;
import java.util.Map;

public class Telemetry {

    public static void track(
            String event,
            Map<String, Object> metadata
    ) {

        System.out.println(
                "[telemetry] " +
                        Instant.now() +
                        " event=" + event +
                        " metadata=" + metadata
        );
    }
}